### ISSUE TITLE
anaconda.org contact page is broken

### ISSUE THREAD

**Issue by stuarteberg** [ContinuumIO/anaconda-issues#8328]
I would like to report a problem with the anaconda.org [contact page](https://anaconda.org/contact).  (I know this is the wrong place to report issues related to anaconda.org -- I'm [supposed](https://github.com/Anaconda-Platform/support) to use the contact page, but since I'm reporting a problem with the contact form itself... well, you see the problem.)

When I attempt to submit a request via the contact form, I see the following error.  I tried both Firefox and Chrome, both logged in and logged out.

<img width="905" alt="image" src="https://user-images.githubusercontent.com/1655821/35486521-19e75894-043d-11e8-9c2b-fb9190516d01.png">

**Comment by abarto** [comment]
You need to click the checkbox in the reCAPTCHA in order for the contact form to work. We'll try to add a more descriptive error message for the field.

**Comment by stuarteberg** [comment]
Thanks for your response.  I, too, wondered if the captcha form was related to the problem, so I made multiple attempts to submit the form, making sure the check the the "I'm not a robot" checkbox every time.  (When the error message is displayed, the captcha box is automatically unchecked again, as shown in the screenshot above.)

Anyway, the issue seems to be magically resolved today.  I was able to use the form successfully today.  Thanks again.
